src/components/Layout.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Toolbar } from '@material-ui/core';
```

Comment:
I usually like to have a Layout component that I can use to setup the page layout, or keep the same layout between multiple pages.